### PR TITLE
Remove DI example code #1987

### LINKED ISSUE #1987: Remove di-examples
Since we have di tests we don't need examples anymore, the code can be deleted.

~~There is example ts-code showing how integration with sidechain through direct invocation can be implemented.
The code is unfortunately not checked every time we make breaking changes to our interfaces, thus very often it doesn't work.
If we want to keep this code we should add relevat jobs to run it on each PR.~~

~~Examples can be run by following commands in tee-worker/ts-tests :~~

~~EVM DI examples: corepack yarn workspace integration-tests evm-di-examples
Substrate DI examples: corepack yarn workspace integration-tests substrate-di-examples~~

### PR DESCRIPTION
**Pull request by songzhouran**
### Context

resolves #1987

### CODE REVIEW

File: tee-worker/docker/lit-substrate-di-test.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      litentry-node:
+        condition: service_healthy
+      litentry-worker-1:
+        condition: service_healthy
+      litentry-worker-2:
+        condition: service_healthy
+    networks:
+      - litentry-test-network
+    entrypoint:
+      "bash -c '/usr/local/worker-cli/lit_ts_test.sh substrate-di-examples 2>&1' "
```
Comment (arieljbirnbaum) [inline tee-worker/docker/lit-substrate-di-test.yml]: This won't work if `substrate-di-examples:staging` isn't defined (and configured with the right `.env` file, etc) in `tee-worker/ts-tests/integration-tests/package.json`.

Not sure if it's worth to add this tbh; the `identity` test suite has already been ported and the `vc` one is [underway](https://github.com/litentry/litentry-parachain/pull/1956); these examples [should become redundant](https://github.com/litentry/litentry-parachain/issues/1987#issuecomment-1665735488) then.

**Review by arieljbirnbaum — CHANGES_REQUESTED** [review]

**Comment by kziemianek** [comment]
As @Kailai-Wang  suggested in issue, let's wait for direct invocation full tests and remove examples so we don't need to care about them anymore.

**Comment by kziemianek** [comment]
Hi @songzhouran what do you think about closing this PR and opening another one with examples removal ?

**Comment by songzhouran** [comment]
@kziemianek I remove code in this PR. Do we need to open another PR?

**Review by kziemianek — APPROVED** [review]

**Comment by kziemianek** [comment]
> @kziemianek I remove code in this PR. Do we need to open another PR?

No, it's fine. 👍🏼

**Review by arieljbirnbaum — APPROVED** [review]
Kill it with fire :raised_hands: :fire:

**Pull request merged**